src/stock-operations/add-stock-operation/add-stock-operation.scss

```diff
@@ -15,3 +11,34 @@ section {
 .modalBody {
   padding-bottom: spacing.$spacing-05;
 }
+
+.actionsContainer {
+  display: "flex";
+  justify-content: space-between;
+  margin: "5px";
+}
+
+.statusDescriptions {
+  display: "flex";
+  flex-direction: "row";
+}
+
+.statusContainer {
+  display: "flex";
+  flex-direction: "row";
+  margin-top: "4px";
```

Comment:
Another thing,we can also adopt carbon spacing for those provided. Like here 4px is `spacing.$spacing-02`